ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/visualization-selector/visualization-selector.view.js

```diff
@@ -0,0 +1,226 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import React from 'react'
+import styled from '../../styles/styled-components'
+const _ = require('underscore')
+
+const CustomElement = styled.div`
+  height: 100%;
+  width: 100%;
+  display: block;
+`
+const Visualization = styled.div`
+  .visualization-choice {
+    white-space: nowrap;
+    padding: ${props => props.theme.largeSpacing}
+    cursor: move;
+    cursor: grab;
+    cursor: -moz-grab;
+    cursor: -webkit-grab;
+  }
+  .visualization-icon {
+    text-align: center;
+    width: ${props => props.theme.minimumButtonSize}
+  }
+
+  .visualization-icon,
+  .visualization-text {
+    display: inline-block;
+    vertical-align: middle;
+  }
+
+  .visualization-text {
+    width: ~'calc(100% - ${props => props.theme.minimumButtonSize})';
+    font-size: ${props => props.theme.mediumFontSize};
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+`
+
+var configs = {
+  openlayers: {
+    title: '2D Map',
+    type: 'component',
+    componentName: 'openlayers',
+    componentState: {},
+  },
+  cesium: {
+    title: '3D Map',
+    type: 'component',
+    componentName: 'cesium',
+    componentState: {},
+  },
+  inspector: {
+    title: 'Inspector',
+    type: 'component',
+    componentName: 'inspector',
+    componentState: {},
+  },
+  table: {
+    title: 'Table',
+    type: 'component',
+    componentName: 'table',
+    componentState: {},
+  },
+  histogram: {
+    title: 'Histogram',
+    type: 'component',
+    componentName: 'histogram',
+    componentState: {},
+  },
+}
+
+function unMaximize(contentItem) {
+  if (contentItem.isMaximised) {
+    contentItem.toggleMaximise()
+    return true
+  } else if (contentItem.contentItems.length === 0) {
+    return false
+  } else {
+    return _.some(contentItem.contentItems, subContentItem => {
```

Comment:
We can throw out `underscore` if we use the Array.some that we polyfill
`return contentItem.contentItems.some(subItem => unmaximize(subItem))`

Although, this seems like it would short-circuit - is that the behavior we want? Is is possible for us to need to un-maximize several children? We should look into this further and see what the behavioral expectations are.